vendor/composer/autoload_classmap.php

```diff
@@ -6,61 +6,27 @@
 $baseDir = dirname($vendorDir);
 
```

Comment:
Sorry for confusion; I meant this entire file.
There are several thousand deletions on in this file ( vendor/composer/autoload_classmap.php ).
I think you likely need to do the following command to fix this:
``````
composer dump-autoload -o
``````

This if from http://www.open-emr.org/wiki/index.php/Composer#Update_Autoloader